spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/RecurrentDecoder.scala

```diff
@@ -121,18 +130,21 @@ class RecurrentDecoder[T : ClassTag](seqLength: Int)
 
   override def backward(input: Tensor[T], gradOutput: Tensor[T]): Tensor[T] = {
```

Comment:
need to implement updateGradInput and accGradParameters too.

Reply:
Added